controllers/machinehealthcheck_targets.go

```diff
@@ -302,7 +302,6 @@ func (r *MachineHealthCheckReconciler) healthCheckTargets(targets []healthCheckT
 		}
 
 		if t.Machine.DeletionTimestamp.IsZero() && t.Node != nil {
```

Comment:
`healthCheckTargets` marks MachineHealthCheckSuccededCondition as false but not as true now can we clarify that in `healthCheckTargets` comment elaborating why.